spec/factories/item.rb

```diff
@@ -3,18 +3,18 @@
 FactoryBot.define do
   factory :item do
     type              { %w[HodinkeeItem CrownandcaliberItem BobswatchesItem].sample }
-    model             { Faker::Commerce.product_name }
     price             { Faker::Commerce.price(range: 0..100_000.0, as_string: true) }
     currency          { %w[usd eur uah].sample }
```

Comment:
suppose we can change `%w[usd eur uah]` to something more dynamic like
`Item.currencies.values` ???